core/src/dpdk/mod.rs

```diff
@@ -144,6 +149,10 @@ impl fmt::Debug for CoreId {
     }
 }
```

Comment:
line doesn't show up here, but the only lcore functionality we're using now (other than `master_lcore` in name) is https://github.com/williamofockham/nb2/blob/a498b66abdbc93192e6700646bccfb96ec99b155/core/src/dpdk/mod.rs#L117, and I'm wondering if that's something we need or is there another function like socket_id by idx that can be used?

Reply:
yeah we should be able to find an equivalent function in `numa.h`. I will take a look and if I find one I can add it to the next pr.